src/debugfs.rs

```diff
@@ -10,7 +10,7 @@ use crate::error::OxidebpfError;
 use crate::LOGGER;
 
 /// Returns the path where `debugfs` is mounted or None if unable to locate.
-pub(crate) fn get_debugfs_mount_point() -> Option<String> {
+pub fn mount_point() -> Option<String> {
```

Comment:
I'm guessing these are rustier because you can call `debugfs::mount_point()?` and `debugfs::mount_if_missing()`?

Reply:
Yep. Even outside of Rust, if the namespace is already called `foo` I don't want to namethings `FooX`, `FooY`, since it should already be clear from the context that we are referring to `foo`. My one exception is traits since those often need to be imported fully for rust to use its functions when doing method resolution.